recipe/meta.yaml

```diff
@@ -163,7 +163,11 @@ requirements:
     # http://doc.qt.io/Qt-5/qdoc-guide-clang.html
     # Only libclang is needed, but needs the full package for detection.
```

Comment:
I think we should consider patching the detection to not require presence of full LLVM, if we only need a single lib that we're packaging separately

Reply:
That sounds like a reasonable improvement. Let me know if you have a good patch for it. Would be glad to incorporate.